rollout/trafficrouting/service_helper.go

```diff
@@ -0,0 +1,38 @@
+package trafficrouting
+
+import (
+	"github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1"
+)
+
+// GetStableAndCanaryServices return a service names for current stable and canary services.
+// If ping-pong feature enabled then the current ping or pong service will be returned. Which is a stable is defined
+// based on a rollout status field Status.Canary.StablePingPong
+func GetStableAndCanaryServices(ro *v1alpha1.Rollout) (string, string) {
+	if IsPingPongEnabled(ro) {
+		canary := ro.Spec.Strategy.Canary
+		if IsStablePing(ro) {
+			return canary.PingPong.PingService, canary.PingPong.PongService
+		} else {
+			return canary.PingPong.PongService, canary.PingPong.PingService
+		}
+	} else {
+		return ro.Spec.Strategy.Canary.StableService, ro.Spec.Strategy.Canary.CanaryService
+	}
+}
+
+func IsStablePing(ro *v1alpha1.Rollout) bool { // reurn "stable is ping"
+	return ro.Status.Canary.StablePingPong == v1alpha1.PPPing
+}
+
+func PingPongOpposite(ro *v1alpha1.Rollout) v1alpha1.PingPongType {
+	//status.stablePingPong == ""
```

Comment:
same here...
can you add a comment block giving different scenarios and  expected return value

Reply:
This method is used in one place only so I've inlined it.